scripts/validate.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 REPO_ROOT=$(git rev-parse --show-toplevel)
-SCHEMA_FILE="${REPO_ROOT}/scripts/io.mediachain.indexer_image_jsonschema_1-0-0.json"
+SCHEMA_FILE="${REPO_ROOT}/scripts/io.mediachain.indexer-image-jsonschema-1-0-0.json"
```

Comment:
Well, it's less offensive than the original, and makes sense if the package name was significant for other than aesthetic reasons.